### PR TITLE
add missing return types in form builder classes

### PR DESCRIPTION
**Pull request by mutec**
fixes 28e142216081467c4d40fa32d730e40b0b101e8a

### CODE REVIEW

File: wcfsetup/install/files/lib/system/form/builder/field/AbstractFormFieldDecorator.class.php [hunk 1]
```diff
@@ -271,31 +271,31 @@ public function cleanup()
     /**
      * @inheritDoc
      */
-    public function checkDependencies()
+    public function checkDependencies(): static
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/system/form/builder/field/AbstractFormFieldDecorator.class.php:274]: This one is not correct.

File: wcfsetup/install/files/lib/system/form/builder/field/acl/AclFormField.class.php [hunk 2]
```diff
@@ -192,8 +192,10 @@ public function readValue()
     /**
      * @inheritDoc
      */
-    public function cleanup()
+    public function cleanup(): static
     {
         ACLHandler::getInstance()->resetValues($this->getObjectType()->objectTypeID);
+
+        return $this;
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/system/form/builder/field/acl/AclFormField.class.php:199]: As this is an actual bug: Can you split this in a separate commit that I can apply independently to 5.5?

File: wcfsetup/install/files/lib/system/form/builder/field/wysiwyg/WysiwygFormField.class.php [hunk 3]
```diff
@@ -113,9 +113,11 @@ public function autosaveId($autosaveId)
     /**
      * @inheritDoc
      */
-    public function cleanup()
+    public function cleanup(): static
     {
         MessageQuoteManager::getInstance()->saved();
+
+        return $this;
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/system/form/builder/field/wysiwyg/WysiwygFormField.class.php:120]: ditto.

**Review by TimWolla — COMMENTED** [review]
Thanks, looking good apart from the few annotations.

**Comment by mutec** [comment]
@TimWolla I just updated it.

**Comment by TimWolla** [comment]
After a rebase on the latest master, only one change remained that I applied with an updated commit message. Thanks.

**Pull request closed without merging**